### PR TITLE
Add type-0 to legacy transaction

### PR DESCRIPTION
**Pull request by jpuri**
Adding type `type: '0x0'` to legacy transaction.

### CODE REVIEW

**Review by Gudahtt — APPROVED** [review]
LGTM!

As a later enhancement, it would be neat if we could control the type used in the transaction, so that we can test what we'd see from a legacy dapp, or one that sends invalid params.

File: src/index.js [hunk 1]
```diff
@@ -319,11 +319,18 @@ const initialize = async () => {
...
-        value: '0x29a2241af62c0000',
-        gasLimit: 21000,
-        gasPrice: 20000000000,
+      const result = await ethereum.request({
+        method: 'eth_sendTransaction',
+        params: [
+          {
+            from: accounts[0],
+            to: '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+            value: '3782DACE9D90000',
```
Comment (Gudahtt) [inline src/index.js:328]: Oh, huh, why the change in `value`?

Comment (jpuri) [reply src/index.js:328]: previous values of 3 eth was emptying my account 😄

Comment (rekmarks) [reply src/index.js:328]: This is a most welcome change! Be advised that it may break our e2e tests, but yeah, 3 ETH is too much, and I always end up editing the amount in practice.

One thing, we should `0x`-prefix it for consistency, and if we're changing it, we might as well go even further to 0.1 ETH, as opposed to the current 0.25:

```suggestion
            value: '0x16345785d8a0000',
```

Comment (jpuri) [reply src/index.js]: @PeterYinusa : please note it may need change in e2e tests for legacy transactions.

Comment (Gudahtt) [reply src/index.js]: That's what I was going to say as well - this isn't a valid parameter without the "0x" prefix.

See here: https://eth.wiki/json-rpc/API#hex-value-encoding
This value is a QUANTITY, the rules for which are described there.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]
